subgraphs/erc20/src/common/constants.ts

```diff
@@ -0,0 +1,21 @@
+import { BigInt, BigDecimal } from "@graphprotocol/graph-ts";
+
+// Constants
+export const REGISTRY_HASH = "QmbPau9knocXzapsM71NfPosujUZRPpTCXpHyVuMNiPWXU"; // data/tokens.json
+
+export const DEFAULT_DECIMALS = 18;
+export const GENESIS_ADDRESS = "0x0000000000000000000000000000000000000000";
+
+export const SECONDS_PER_DAY = 60 * 60 * 24;
+export const SECONDS_PER_HOUR = 60 * 60;
+
+export const BIGINT_ZERO = BigInt.fromI32(0);
+export const BIGINT_ONE = BigInt.fromI32(1);
+export const BIGINT_TWO = BigInt.fromI32(2);
+export const BIGDECIMAL_ZERO = new BigDecimal(BIGINT_ZERO);
+export const BIGDECIMAL_ONE = new BigDecimal(BIGINT_ONE);
+
+// Flags
+export const DETAILED_TOKEN = 1 << 0;
```

Comment:
Where is this used?

Reply:
It was designed to be a flag to indicate whether an erc20 token supports functions such as name(), symbol(). After consideration, I decided to disable this flag as when there are several hundred erc20 tokens to index there is no way to     know it in advance and we just try to call these methods later.